Neos.Flow/Classes/Mvc/DispatchComponent.php

```diff
@@ -34,7 +35,7 @@ class DispatchComponent implements ComponentInterface
      */
     public function handle(ComponentContext $componentContext)
     {
-        $actionRequest = $componentContext->getParameter(DispatchComponent::class, 'actionRequest');
+        $actionRequest = $componentContext->getHttpRequest()->getAttribute(ServerRequestAttributes::ACTION_REQUEST);
```

Comment:
A little
```
if ($actionRequest === null) {
  throw meaningfullexception..
}
```
would be nice here in case the MW chain is configured wrongly or something else went wrong

Reply:
That thing will be gone with the follow-up PR anyway :)